tests/test_repositories.py

```diff
@@ -76,7 +68,7 @@ def test_create_local_repository_using_create_repo_fail_if_user_already_exists(m
     with pytest.raises(RepositoryAlreadyExistsException):
         artifactory_repo.create_repo(LOCAL_REPOSITORY)
 
-        artifactory_repo.create_local_repo.assert_called_once_with(LOCAL_REPOSITORY)
+    artifactory_repo.get_repo.assert_called_once_with(LOCAL_REPOSITORY)

```

Comment:
Test is not passing because you are spying `create_local_repo` but are asserting that `get_repo` is called. Do you really want to look at `get_repo`? If yes you need to change the `mocker.spy`, and I think that this function is called with `LOCAL_REPOSITORY.key`.